src/main/java/com/adyen/model/balanceplatform/PaymentInstrument.java

```diff
@@ -52,13 +52,14 @@
 
 public class PaymentInstrument {
   public static final String JSON_PROPERTY_ADDITIONAL_BANK_ACCOUNT_IDENTIFICATIONS = "additionalBankAccountIdentifications";
+  @Deprecated
   private List<PaymentInstrumentAdditionalBankAccountIdentificationsInner> additionalBankAccountIdentifications = null;
 
   public static final String JSON_PROPERTY_BALANCE_ACCOUNT_ID = "balanceAccountId";
   private String balanceAccountId;
 
   public static final String JSON_PROPERTY_BANK_ACCOUNT = "bankAccount";
-  private PaymentInstrumentBankAccount bankAccount;
+  private BankAccountDTO bankAccount;
```

Comment:
@wboereboom `BankAccountDTO` they should change this name right, external resources should not be called like this.

Reply:
Fixed!